lib/client.js

```diff
@@ -15,7 +15,7 @@ const { through } = require('./utils')
 
 module.exports = Client
 
-function Client (broker, conn, req) {
+function Client (broker, conn, req = null) {
```

Comment:
```suggestion
function Client (broker, conn, req) {
```

Reply:
as mentioned in the docs, this is a fix for the optional parameter which is only used for websockets.

Reply:
@haci-xplora I know that but there is still no reason to set it to null as it will be undefined when omitted